README.md

```diff
@@ -28,29 +28,27 @@
 ## Feature Sprint 
 - We use feature-based sprints, [tracked here](https://github.com/orgs/openfoodfacts/projects/83)
 
-## Alert!
-
-We are currently using Flutter 3.0.5 as the new 3.3.0 [has some bugs](https://github.com/openfoodfacts/smooth-app/issues/2919).
-
-Running `flutter downgrade 3.0.5` downgrades the version.
-
-------
```

Comment:
Why did you remove this, thats important

Reply:
Because it seems we're now on 3.3.0 if I'm not mistaken ?

Reply:
ok, I talked too fast: https://github.com/openfoodfacts/smooth-app/blob/develop/packages/smooth_app/pubspec.yaml#L8